connect/runtime/src/test/java/org/apache/kafka/connect/runtime/distributed/DistributedHerderTest.java

```diff
@@ -917,6 +955,7 @@ public void testDestroyConnector() throws Exception {
         PowerMock.verifyAll();
     }
 
+    @Disabled("Disabled until EasyMock->Mockito conversion is done")
```

Comment:
We cannot disable tests like this.

Reply:
This is only temporary, as a way to keep all the source code in place while iterating on the file (since it's a large file).  I will be removing the annotation one-by-one as I fix each case.  Just wanted to validate the basic approach before going to all the trouble.